src/Pages/Signup/Signup.jsx

```diff
@@ -0,0 +1,163 @@
+  };
+
+  const passwordHandler = () => {
+    if (showPassword === "text") {
+      setShowPassword("password");
+      setIcon("visibility_off");
+    } else {
+      setShowPassword("text");
+      setIcon("visibility");
+    }
+  };
+
+  return (
+    <>
+      <Header />
+      <main className="login signup">
+        <div className="login-container">
+          <h3 className="center-text">Signup</h3>
+          <div className="input-group">
+            <label className="form-label">First Name</label>
+            <input
+              className="form-control"
+              type="text"
+              name="firstName"
+              value={user.firstName}
+              onChange={userInputHandler}
+              placeholder="Enter Email"
+            />
+          </div>
+          <small style={{color:color}}>{errorMsg && "Enter first name"}</small>
+          <div className="input-group">
+            <label className="form-label">Last Name</label>
+            <input
+              className="form-control"
+              type="text"
+              name="lastName"
+              value={user.lastName}
+              onChange={userInputHandler}
+              placeholder="Enter Email"
+            />
+          </div>
+          <small style={{color:color}}>{errorMsg&&"Enter last name"}</small>
+          <div className="input-group">
+            <label className="form-label">{"Email last name"}</label>
+            <input
+              className="form-control"
+              type="email"
+              name="email"
+              onChange={userInputHandler}
+              value={user.email}
+              placeholder="Enter Email"
+            />
+          </div>
+          <small style={{color:color}}>{ errorMsg&&"Enter email"}</small>
+          <div className="input-group">
+            <label className="form-label">Password</label>
+
+            <span
+              onClick={passwordHandler}
+              className="material-icons visibility"
+            >
+              {icon}
+            </span>
+
+            <input
+              className="form-control"
+              type={showPassword}
+              name="password"
+              onChange={userInputHandler}
+              value={user.password}
+              placeholder="Enter Password"
+            />
+          </div>
+          <small style={{color:color}}>{errorMsg&&"Enter password "}</small>
+          <div className="checkbox-parent">
+            <input onChange={checkboxHandler} type="checkbox" />
+            <label>I accept all Terms & Conditions</label>
+          </div>
+
+          <div className="center">
+            <button onClick={signupHandler} className="my-2 login-btn primary-bg">
+              Create New Account
+            </button>
+          </div>
+          <div className="center">
+            <Link className="black-text" to="/login">
+              Already have an account
+            </Link>
+            <svg
```

Comment:
you can host the svg and use it for clean code

Reply:
ok